src/gui/Gui.cc

```diff
@@ -457,6 +457,9 @@ std::unique_ptr<gz::gui::Application> createGui(
                 "</gz-gui>"));
             }
 
+            msg += " SDF code to replace GzScene3D is available at " +
+              " https://github.com/gazebosim/gz-sim/blob/gz-sim7/Migration.md\n"
```

Comment:
I would add this to the the string above, where `msg` is being constructed (lines 440-442). Also, instead of printing a link, I would just say "see migration guide for more details" but this is only my opinion. 

For example, lines 440-442:
```c++
            std::string msg{
                "The [GzScene3D] GUI plugin has been removed since Garden "
                "(see migration guide for more details). "
                "Loading the following plugins instead:\n"};
```

Reply:
I've moved the message to that lines in [8325cf5](https://github.com/gazebosim/gz-sim/pull/1698/commits/8325cf57cce0b435b48c52b84231330a8389e9df). I would prefer to facilitate people the link so migration can work just by clicking, we (developers) tend to be lazy. Not strong on this if you really prefer to avoid URLs.

Reply:
I'm not strong on this either so sounds good to me